common/presenters/move-to-meta-list-component.js

```diff
@@ -68,6 +74,14 @@ function moveToMetaListComponent({
         text: filters.formatTime(timeDue),
       },
     },
+    {
```

Comment:
It feels like this should actually be instead of the existing date item. Right now we could have the following items all displayed at once:

- Date
- Time due
- Date of travel

It feels like what we need to actually do with this change is allow the existing Date row to support a date range as well as a single value.

And we can check with @Andy-Millar if he wants to update that content to `Date of travel` from what we already have which is just `Date`. Either way I think the label should be consistent between the two scenarios:

![image](https://user-images.githubusercontent.com/3327997/79270520-3fdf1480-7e96-11ea-8518-b7b5548ce042.png)

Reply:
I have confirmed with @Andy-Millar that date is fine. I was operating under the possibly wrong assumption that the range would go away once the date is fixed. I will change it to prioritise the fixed date if present, and to display the range if not present.